lib/api/webSockets.js

```diff
@@ -58,12 +58,19 @@ function connect(connectionOps = {}) {
 		}
 
 		if (content.type === 'console' && content.data) {
-			const method = {
-				info: 'info',
-				log: 'log',
-			}[content.subtype] || 'debug';
-
-			return logger[method](`|L| ${method}:`, content.data.join('\n'));
+			const logType = {
+				info: '|AI|',
+				log: '|AL|',
+				warn: '|AW|',
+				error: '|AE|',
+			}[content.subtype] || '|AL|';
+
+			// ADDING logType after new line characters needed for proper displaying multilines
+			// TODO: need to remove replacing after release test-repl-with-updated-logs branch
+			return logger.log(
+				logType,
+				content.data.join('\n').replace(/\n/g, `\n${logType} `)
```

Comment:
if `content.data` is an array, you could have written it just as `...content.data`

Reply:
and only first line of multi-line log will be properly marked. We decide yesterday that we need insert markers after each new lines before branch test-repl-with-updated-logs will be release.

Reply:
i thought it did not work with multiline string but *did* work with multiple arguments, but I guess it does not.
Anyway then `content.data.forEach(one => logger.log(logType, one))` could still be a better option to string replacing.

Reply:
I believe that this is not crucial point, anyway it will be changed after repl release.

Reply:
there is arrToString helper somewhere

Reply:
I don't mind if we leave that, repl is in the late stage of its backing anyway